### DISCUSSION TITLE
Download link?

### DISCUSSION THREAD

**Discussion by TheRamenChef** [optimatika/ojAlgo#476] [General, ojAlgo and related code]
This feels like a dumb question, but… where can I actually download the library? I can't find anything on the website or the repository that looks like a download link. Am I just supposed to download the source code from the repo?

**Comment by apete** [comment]
I'd recommend you to use Maven, Gradle or any other dependency Management tool.

Even if you don't, you may get the individual jars from their repositories.

https://mvnrepository.com/artifact/org.ojalgo/ojalgo
